.github/workflows/openssf-scorecards.yaml

```diff
@@ -0,0 +1,71 @@
+name: Scorecards supply-chain security
+
+# Currently this workflow is failing and should be run only when triggered manually
+# If workflow result is green:
+#    - remove `workflow_dispatch` and uncomment remaining lines 
+on:
+  workflow_dispatch:  
```

Comment:
let's do it in multiple steps:
1. trigger on `push` to `master` and merge
   * does not block normal CI
   * shows issues on `master`
2. merge fixes to `master`
3. trigger openssf action on `master` and `pull_request`
   * no more issues on `master`
   * can't merge pull request with openssf issues anymore